src/bp/core/services/admin/community.ts

```diff
@@ -122,7 +124,7 @@ export class CommunityAdminService implements AdminService {
       )
   }
 
-  async addBot(teamId: number, bot: Bot, botTemplate?: BotTemplate): Promise<void> {
+  private async _addBot(teamId: number, bot, botTemplate?: BotTemplate): Promise<void> {
```

Comment:
This method is called by each node, it will fail since they share the same postgres db and they will all try to insert the bot